group_vars/all.yml

```diff
@@ -9,6 +9,9 @@ default_archive_base_port: 6500
 default_publishing_base_port: 6600
 default_authoring_base_port: 6400
 
+# See http://docs.ansible.com/ansible/postgresql_user_module.html
+# Change in group_vars or host_vars for environment specific settings
+role_attr_flags: CREATEDB,NOSUPERUSER
```

Comment:
I'm not exactly sure this is going to be enough since cnx's database creation logic likes to be a superuser, but we'll roll with it. If it fails later, we'll have at least tried. 👍